src/firstvoices/backend/admin/dictionary_admin.py

```diff
@@ -64,3 +64,7 @@ class DictionaryEntryHiddenBaseAdmin(HiddenBaseAdmin):
         AlternateSpellingInline,
         PronunciationInline,
     ]
+
+
+class PartsOfSpeechAdmin(BaseAdmin):
+    list_display = ("title",) + BaseAdmin.list_display
```

Comment:
The "parent" field could be added here.

Reply:
Hmm, I'm not sure why but I can still see and am able to edit the parent field without mentioning it here. I've also tried hard reloading to eliminate the cache possibility on browser side, is there any other reason why "parent" field is still visible without mentioning here ?
I've added it explicitly now though, but just wondering.

Reply:
The "list_display" attribute only affects the list view in the admin panel (eg: "/admin/backend/partofspeech/"). If the "fields" attribute is not given here then for the create and edit pages all fields will be displayed.